sources/packages/backend/libs/sims-db/src/entities/disbursement-overaward.model.ts

```diff
@@ -97,4 +100,31 @@ export class DisbursementOveraward extends RecordDataModel {
     nullable: true,
   })
   deletedAt?: Date;
+  /**
+   * Note entered during manual overaward record added.
+   */
+  @OneToOne(() => Note, { eager: false, cascade: true })
+  @JoinColumn({
+    name: "note_id",
+    referencedColumnName: ColumnNames.ID,
+  })
+  overawardNotes: Note;
+
```

Comment:
Please follow the other file spacing pattern and remove the blank lines, lines 112 and 122.